src/hotspot/share/gc/shenandoah/shenandoahMmuTracker.hpp

```diff
@@ -50,16 +50,32 @@ class ShenandoahMmuTask;
  * MMU.
  */
 class ShenandoahMmuTracker {
+private:
+  // For reporting utilization during most recent GC cycle
+  double _most_recent_timestamp;
+  double _most_recent_gc_time;
+  double _most_recent_gcu;
+  double _most_recent_mutator_time;
+  double _most_recent_mu;
+
+  // For periodic MU/GCU reports
+  double _most_recent_periodic_time_stamp;
+  double _most_recent_periodic_gc_time;
+  double _most_recent_periodic_mutator_time;
+
+  uint _most_recent_gcid;
+  uint _active_processors;
 
-  double _generational_reference_time_s;
-  double _process_reference_time_s;
-  double _collector_reference_time_s;
+  bool _most_recent_is_full;
+  bool _doing_mixed_evacuations;
```

Comment:
This is written, but never read. Is this used somewhere on a different branch as part of the bigger picture?

Reply:
Good catch.  It is no longer used.  I'll remove it.  (Its a remnant of early experiments with using GCU to trigger GC and wanting to treat GCU during mixed evac as different than GCU outside of mixed evac)  I couldn't figure out how to get a good trigger from GCU so I'll remove this for now.